README.md

```diff
@@ -230,6 +230,43 @@ jobs:
 ```
 </details>
 
+## Deleting caches
+We can not re-use caches from pull request branches in other branches like main, such caches can eat up the storage quota and hence causing thrashing on more useful branches like main. In order to resolve this issue, we can use [gh-actions-cache cli](https://github.com/actions/gh-actions-cache/) to delete caches. This workflow uses `gh-actions-cache` to delete all the caches created by all the pull requests. 
```

Comment:
>This workflow uses `gh-actions-cache` to delete all the caches created by all the pull requests.

IMO this is just an example. Users can create similar other workflows for other cases like feature branch etc. Hence we should call this out as an example. 
We can also think if we want to provide more examples.